features/net/network-socket/nsapi_dns.cpp

```diff
@@ -19,21 +19,29 @@
 #include <string.h>
 #include <stdlib.h>
 
+#define CLASS_IN 1
+
+#define RR_A 1
+#define RR_AAAA 28
 
 // DNS options
-#define DNS_BUFFER_SIZE 256 
+#define DNS_BUFFER_SIZE 512
```

Comment:
Just checking, was there a test that confirmed the original buffer size was too small?